osclib/list_command.py

```diff
@@ -59,7 +59,7 @@ def perform(self, supersede=False):
                 if message:
                     line += '\n' + Fore.WHITE + message + Fore.RESET
 
-                print(' ' + line)
+                print('  ' + line)
```

Comment:
`print(' ', line)` is more natural here IMHO (two spaces and not very visible)

Reply:
Except that the output is then:

```
(' ', '$line')
```

Reply:
not in python3 nor with the future:

```
>>> from __future__ import print_function
>>> line='hallo'
>>> print(' ', line)
  hallo
```

(which if you checked, the list_comand.py already imports)

Reply:
Indeed, what @coolo says.

Reply:
Doesn't bother me either way.